### PR TITLE
Add Conference Schedule Page

### LINKED ISSUE #184: Conference Schedule Page
Similar to #181, but showing the schedule in full-width on the conference page. This is probably more relevant for multi-track/multi-day conference than it is for single-track/single-day conferences.

Conference Show Schedule@2x

### PR DESCRIPTION
**Pull request by marcoroth**
This pull request implements a page for showing the conference schedule for conferences that have a `schedule.yml` file in their data folder.

This pull request and design is a draft/work in progress.

CleanShot 2024-10-20 at 06 34 23

 
Resolves #184

### CODE REVIEW

**Comment by marcoroth** [comment]
CleanShot 2024-10-20 at 07 08 36

**Comment by marcoroth** [comment]
CleanShot 2024-10-20 at 07 55 53

**Comment by adrienpoly** [comment]
This looks awesome. 
It does look also very time consuming to construct. I guess that once it is done , it is done. Do you think we will be able to somehow automate this ?

**Comment by marcoroth** [comment]
I started to order all talks in running order. The idea here is that if the  talks are already in running order in the videos.yml file you only have to "construct" the grid in the schedule.yml file.

Also, this is designed to be gradual. If the conference doesn't have a schedule.yml it doesn't try to display it.

It's a bit more effort per conference, but I think it's worth it.

In the big Picture: if we can use this app to power conference schedules not only for past conferences, but also for current and future ones then this feature becomes necessary.

File: test/fixtures/organisations.yml [hunk 1]
```diff
@@ -54,7 +54,7 @@ rails_world:
   frequency: 1
   youtube_channel_id: UC9zbLaqReIdoFfzdUbh13Nw
   youtube_channel_name: railsofficial
-  slug: railsworld
+  slug: rails-world
```
Comment (adrienpoly) [inline test/fixtures/organisations.yml:57]: We probably need to do something manually in prod for that change

Comment (marcoroth) [reply test/fixtures/organisations.yml:57]: I actually already changed that months ago:

https://www.rubyvideo.dev/organisations/rails-world

**Comment by marcoroth** [comment]
Merging this for now.

CleanShot 2024-12-08 at 17 47 22

**Comment by marcoroth** [comment]
image

**Comment by marcoroth** [comment]
Advanced item rendering:

**Comment by marcoroth** [comment]
Single track:

CleanShot 2024-12-08 at 17 51 27

**Pull request merged**